frontend/src/app/shared/dialog/confirm-dialog/confirm-dialog.component.ts

```diff
@@ -11,44 +12,13 @@ export class ConfirmDialogComponent implements OnInit {
   dialogTitle: string = '';
   dialogText: string = '';
   constructor(
-    @Inject(MAT_DIALOG_DATA) public data: any,
+    @Inject(MAT_DIALOG_DATA) public data: ConfirmDialogData,
     public dialogRef: MatDialogRef<ConfirmDialogComponent>,
-    private translate: TranslateService,
   ) {}
 
   ngOnInit() {
-    if (this.data.draftCreate) {
-      this.dialogTitle = 'Check-in im Draft-Status';
-      this.dialogText =
-        'Dein Objective befindet sich noch im DRAFT Status. Möchtest du das Check-in trotzdem erfassen?';
-    } else if (this.data.action) {
-      if (this.data.action === 'release') {
-        this.dialogTitle = this.data.title + ' veröffentlichen';
-        this.dialogText = 'Soll dieses ' + this.data.title + ' veröffentlicht werden?';
-      } else if (this.data.action === 'todraft') {
-        this.dialogTitle = this.data.title + ' als Draft speichern';
-        this.dialogText = 'Soll dieses ' + this.data.title + ' als Draft gespeichert werden?';
-      }
-    } else {
-      this.dialogTitle = this.data.title + ' löschen';
-      if (this.data.isAction) {
-        this.dialogText = 'Möchtest du diese Action wirklich löschen?';
-      } else {
-        let error;
-        switch (this.data.title) {
-          case 'Team':
-            error = 'DELETE_TEAM';
-            break;
-          case 'Objective':
-            error = 'DELETE_OBJECTIVE';
-            break;
-          case 'Key Result':
-            error = 'DELETE_KEY_RESULT';
-            break;
-        }
-        this.dialogText = this.translate.instant('INFORMATION.' + error);
-      }
-    }
+    this.dialogTitle = this.data.title || 'Are you sure?';
```

Comment:
update confirm logic and translations

Reply:
Will be fixed in pr #1072 
And behavior will be checked by [this](https://github.com/puzzle/okr/pull/1072/files#diff-f0558a6e2f00600d7f3a244322d44962271ef8d2369b1962e773f38c5e9ae642R31)